infrastructure/eks/networking.tf

```diff
@@ -0,0 +1,134 @@
+# VPC Configuration
+resource "aws_vpc" "main" {
```

Comment:
I hate to bring you this after all the work you've done, but there's a good terraform module you could have used for this 
https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest